libavfilter/vf_drawbox.c

```diff
@@ -85,6 +85,61 @@ typedef struct DrawBoxContext {
 
 static const int NUM_EXPR_EVALS = 5;
```

Comment:
IMO, it is not needed: 
Make code more concise and suitable for following supporting box source
from side data.